src/base/static/components/utils/scroll-helpers.js

```diff
@@ -0,0 +1,20 @@
+// Due to https://stackoverflow.com/questions/8917921/cross-browser-javascript-not-jquery-scroll-to-top-animation
+const scrollTo = (elt, to, duration) => {
+  const difference = to - elt.scrollTop;
+  const perTick = difference / duration;
+  setTimeout(() => {
+    elt.scrollTop = elt.scrollTop + perTick;
+    if (elt.scrollTop === to) return;
+    scrollTo(elt, to, duration - 10);
+  }, 10);
```

Comment:
Can we refactor these to use `requestAnimationFrame` instead of `setTimeout` here?